src/main/java/frc/team3388/robot/IterativeRobot.java

```diff
@@ -1,14 +1,37 @@
 package frc.team3388.robot;
 
+import com.ctre.phoenix.motorcontrol.FeedbackDevice;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
 import com.flash3388.flashlib.frc.robot.IterativeRobotInterface;
 import com.flash3388.flashlib.robot.Robot;
+import com.flash3388.flashlib.robot.hid.Button;
+import com.flash3388.flashlib.robot.hid.xbox.XboxButton;
+import com.flash3388.flashlib.robot.hid.xbox.XboxController;
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj.SpeedController;
+import frc.team3388.robot.actions.shooter.ShootShooter;
+import frc.team3388.robot.subsystems.Shooter;
 
 public class IterativeRobot implements IterativeRobotInterface {
 
     private final Robot mRobot;
+    private final Shooter shooter;
+    private final XboxController xboxController;
 
     public IterativeRobot(Robot robot) {
         mRobot = robot;
+
+        WPI_TalonSRX speedController = new WPI_TalonSRX(RobotMap.SHOOTER_CONTROLLER);
+        configSpeedController(speedController);
+
+        DoubleSolenoid doubleSolenoid = new DoubleSolenoid(RobotMap.DOUBLE_SOLENOID_1, RobotMap.DOUBLE_SOLENOID_2);
+        EncoderSrx encoderSrx = new EncoderSrx(speedController, false);
+        shooter = new Shooter(speedController, doubleSolenoid, encoderSrx);
```

Comment:
I would suggest using a factory method `standard()` or something that will create this subsystem with the default parameters instead of creating the objects from outside